monitoring/src/deposit-monitor.ts

```diff
@@ -7,30 +7,34 @@ import type { SystemEvent, Monitor as SystemEventMonitor } from "./system-event"
 import type { DepositRevealedEvent as DepositRevealedChainEvent } from "@keep-network/tbtc-v2.ts/dist/src/deposit"
 import type { Bridge } from "@keep-network/tbtc-v2.ts/dist/src/chain"
 
+function satsToRoundedBTC(sats: BigNumber): string {
+  return (sats.div(BigNumber.from(1e6)).toNumber() / 100).toFixed(2)
+}
+
 const DepositRevealed = (
   chainEvent: DepositRevealedChainEvent
 ): SystemEvent => ({
-  title: "Deposit revealed",
+  title: "Deposit Revealed",
   type: SystemEventType.Informational,
   data: {
-    btcFundingTxHash: chainEvent.fundingTxHash.toString(),
+    btcFundingTxHash: `https://mempool.space/tx/${chainEvent.fundingTxHash.toString()}`,
```

Comment:
Remember that we have both testnet and mainnet so we need to use the correct links depending on the network. You can take the environment from here: https://github.com/keep-network/tbtc-v2/blob/main/monitoring/src/context.ts. Same comment to all links in this file.

Reply:
[e953b0](https://github.com/keep-network/tbtc-v2/pull/612/commits/e953b05feda4f54b0864ecf41e0b7d7c4aaf6e26)